app/views/admin/places/_form.html.erb

```diff
@@ -1,37 +1,173 @@
 <%= render :partial => 'admin/services/overview' %>
 
-<%= semantic_form_for [:admin, @service, @data_set, @place], html: {class: 'well'} do |f| %>
+<%= form_for [:admin, @service, @data_set, @place], html: {class: 'well'} do |f| %>
 
-  <%= f.semantic_errors :source_address, :class => "alert alert-danger" %>
+  <div class="form-group">
+    <span class="form-label">
+      <%= f.label :name %>
+    </span>
+    <span class="form-wrapper">
+      <%= f.text_field :name, class: 'input-md-6 form-control' %>
+    </span>
+  </div>
 
-  <%= f.input :name, :input_html => { :size => 60, :class => 'input-md-6' } %>
+  <%= f.fields_for "Address" do %>
+    <legend>
+      <span>Address</span>
+    </legend>
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :address1 %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :address1, class: 'input-md-6 form-control' %>
+      </span>
+    </div>
 
-  <%= f.inputs "Address" do %>
-    <%= f.input :address1, :label => "Address 1", input_html: {class: 'input-md-6'} %>
-    <%= f.input :address2, :label => "Address 2", input_html: {class: 'input-md-6'} %>
-    <%= f.input :town, input_html: {class: 'input-md-6'} %>
-    <%= f.input :postcode, input_html: {class: 'input-md-2'} %>
-    <%= f.input :override_lng, :label => "Override longitude", :placeholder => "Leave blank to use postcode coords", input_html: {class: 'input-md-4'}%>
-    <%= f.input :override_lat, :label => "Override latitude", :placeholder => "Leave blank to use postcode coords", input_html: {class: 'input-md-4'} %>
-    <%= f.input :snac, :label => "SNAC code", input_html: {class: 'input-md-2'} %>
-  <% end %>
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :address2 %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :address2, class: 'input-md-6 form-control' %>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :town %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :town, class: 'input-md-6 form-control' %>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :postcode %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :postcode, class: 'input-md-2 form-control' %>
+        <% if f.object.errors[:postcode].any? %>
+          <div class="error-text" id="place_override_postcode_input">
+            <%= f.object.errors[:postcode].to_sentence %>
+          </div>
```

Comment:
It seems formtastic added this automatically somehow... If there's a better way of doing this please let me know

Reply:
Is there a default class we could use instead of writing our own css like I see `f.semantic_errors :source_address, :class => "alert alert-danger"`. Maybe one of those alert classes does what error-text does?

Reply:
There was a handy bootstrap class 👍